.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,22 @@
+# Description
+
+<!-- Please describe the changes included in this PR here. -->
+
+# To-Dos
+
+<!-- Before submitting this PR, please lint and build your changes locally. -->
+<!-- Add an 'x' between the brackets to mark each checkbox as checked. -->
+
+- [ ] Test coverage for new code
```

Comment:
Somewhat unclear to me. Maybe something like:

- [ ] Added or changed functionality is covered by tests

wdyt?

Reply:
How about:

```
- [ ] Test coverage added for new or updated functionality
```

Reply:
Like it